test.test.ts

```diff
@@ -0,0 +1,61 @@
+import { describe, it } from 'node:test';
```

Comment:
Used built-in Node.js test runner and assert to reduce dependencies and improve execution performance.